### DISCUSSION TITLE
A test :world_map:

### DISCUSSION THREAD

**Discussion by emmaviolet** [emmaviolet/just-a-readme#13] [Ideas]
:world_map:

**Comment by emmaviolet** [comment]
cc @emmaviolet

**Comment by emmaviolet** [comment]
@entcheva how does this discussion notification look to you?

**Comment by entcheva** [comment]
Can confirm the discussion notification does not render emoji: 

<img width="256" alt="Screen Shot 2022-11-07 at 11 53 39 AM" src="https://user-images.githubusercontent.com/1369728/200368455-4e68aa4c-fbd9-4888-896d-37cb658fbf7f.png">
